app/__init__.py

```
@@ -1,9 +1,27 @@
+import os
+
 from flask import Flask
 
-def create_app():
-    app = Flask(__name__)
+def create_app(test_config=None):
+    app = Flask(__name__, instance_relative_config=True)
+    app.config.from_mapping(
+            SECRET_KEY='dev',
+            DATABASE=os.path.join(app.instance_path, 'skemaDB.sqlite')
+    )
+    if test_config is None:
+        app.config.from_pyfile('config.py', silent=True)
+    else:
+        app.config.from_mapping(config)
+
+    try:
+        os.makedirs(app.instance_path)
+    except OSError:
+        pass
```

What does this error mean? Should we maybe be logging it somehow at least? It just strikes me as wrong to catch and ignore an error

---

An OSError is thrown when using the cli with flask.  It's from the OS package at the top of the file.  I could add in some sort of log.